### DISCUSSION TITLE
[Improvement][E-Commerce]: Add folder structure to pricing rules overview

### DISCUSSION THREAD

**Discussion by foxgreek** [pimcore/pimcore#12126] [General]
### Improvement description

We are planning to use the pricing rules for individual distributors. So we will have more than 50000 rules.

For performance reasons and usability, is it possible to add the folder structure from the user management to the pricing rules overview?

Or do you recommend another approaches for individual pricing rules?

Thanks a lot!

**Comment by dvesh3** [comment]
@foxgreek I moved the issue to Discussions, which is the right place to ask questions or getting help. thanks!
